src/core/lib/slice/slice_internal.h

```diff
@@ -46,4 +46,9 @@ grpc_slice grpc_slice_maybe_static_intern(grpc_slice slice,
 uint32_t grpc_static_slice_hash(grpc_slice s);
 int grpc_static_slice_eq(grpc_slice a, grpc_slice b);
 
+// Returns the memory used by this slice, not counting the slice structure
```

Comment:
Why not include the size of the slice structure itself?  We do actually allocate memory for that, don't we?

Reply:
The slice struct is held within the TraceEvent class, so that memory is counted when we do sizeof(TraceEvent)